applicationintegration/applicationintegration_edit_region/main.tf

```diff
@@ -0,0 +1,50 @@
+/**
+* Copyright 2024 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+data "google_project" "test_project" {
+}
+
+resource "google_kms_key_ring" "keyring" {
+  name     = "my-keyring"
+  location = "us-central1"
+}
+
+resource "google_kms_crypto_key" "cryptokey" {
+  name            = "crypto-key-example"
+  key_ring        = google_kms_key_ring.keyring.id
+  rotation_period = "7776000s"
+  depends_on      = [google_kms_key_ring.keyring]
+}
+# [START applicationintegration_edit_region]
```

Comment:
is this the right region tag prefix?

Reply:
just verified, it should be application_integration

Reply:
Thanks! I've fixed the region tag in https://github.com/terraform-google-modules/terraform-docs-samples/pull/637/commits/98d1c3103c72bc44bf40cd0d5e10d91f18ab181d.